src/modules/position-creator/services/position.creator.transaction.ts

```diff
@@ -740,7 +787,49 @@ export class PositionCreatorTransactionService {
             );
         }
 
-        return interaction.buildTransaction().toPlainObject();
+        return [interaction.buildTransaction().toPlainObject()];
+    }
+
+    private async getMinimumAmountsForLiquidity(
+        swapRoute: SwapRouteModel,
+    ): Promise<BigNumber[]> {
+        const pair = swapRoute.pairs[0];
+        let [amount0, amount1] =
```

Comment:
same here, i think if is easier to be followed

Reply:
done